addons/safestart/config.cpp

```diff
@@ -6,7 +6,8 @@ class CfgPatches {
         weapons[] = {};
         requiredVersion = REQUIRED_VERSION;
         requiredAddons[] = {
-            "afm_main"
+            "afm_main",
+            "afm_common"
```

Comment:
If `afm_common` requires `afm_main` isn't it enough to require `afm_common` only?

Reply:
It is. 👍